src/Couchbase.Lite.Tests.Shared/ReplicationTest.cs

```diff
@@ -70,9 +70,35 @@ public ReplicatorTest()
             ReopenDB();
             Database.Delete("otherdb", Directory);
             _otherDB = OpenDB("otherdb");
+            Database.SetLogLevel(LogDomain.Replicator, LogLevel.Verbose);
         }
 
-        #if COUCHBASE_ENTERPRISE
+        [Fact]
+        public async Task TestCloseServerWhenReplicatorExistsAsync()
+        {
+            var targetEndpoint = new URLEndpoint(new Uri("ws://192.168.0.11:4984/app"));
+            var config = new ReplicatorConfiguration(Db, targetEndpoint);
+            using (var repl = new Replicator(config))
+            {
+                repl.Start();
+                var count = 0;
+
+                this.Invoking(x => ReopenDB())
+                    .ShouldThrow<CouchbaseLiteException>(
+                        "because the database cannot be closed while replication is running");
```

Comment:
This assertion is not essential to the test, but there is no harm in leaving it in